perllib/FixMyStreet/App/Controller/Waste.pm

```diff
@@ -1264,7 +1276,10 @@ sub process_garden_data : Private {
     $data->{bin_count} = $bin_count;
     $data->{new_bins} = $data->{bins_wanted} - $data->{current_bins};
 
-    my $total = $c->cobrand->garden_waste_cost($bin_count);
+    my $cost_pa = $c->cobrand->garden_waste_cost_pa($bin_count);
+    my $cost_now_admin = $c->cobrand->garden_waste_new_bin_admin_fee($data->{new_bins});
+    my $total = $cost_now_admin + $cost_pa;
+
     $c->set_param('payment', $total);
```

Comment:
I think we need to store the total, the admin and the annual fee as the latter two need to be broken out in the Kingston CC parameters.

Reply:
Yep, have done that now in 9d5ff70bdb4e7f410738fe0e7df9e50f892c69da